commerce-price-list-service/src/main/java/com/liferay/commerce/price/list/service/impl/CommercePriceListLocalServiceImpl.java

```diff
@@ -84,7 +86,7 @@
 
 	@Override
 	public CommercePriceList addCommercePriceList(
-			long groupId, long userId, long commerceCurrencyId,
+			long groupId, long userId, String commerceCurrencyCode,
```

Comment:
We could avoid a major version increase by just deprecating these existing method signatures, adding new ones, and having the old methods call the new ones -- since it should be easy to derive the `commerceCurrencyCode` given a `commerceCurrencyId`. But I'm not sure about the priority on this, @marco-leo .

Reply:
Yes but, since currencyId was in all of the methods, it would mean that I need to duplicate all of them and reference the new ones from the olds

Reply:
Yeah, that's true :/ -- it'd be a lot of new methods. Just depends on how much we care about the major version increase, I guess.